src/styles/index.scss

```diff
@@ -1,3 +1,153 @@
+$font-family-primary: 'Roboto', Arial, sans-serif;
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: 400px;
```

Comment:
The `font-weight` property should be specified without units. It should be just a number, not a number with 'px'.